nsis/gvim_version.nsh

```diff
@@ -3,4 +3,5 @@
 !define __GVIM_VER__NSH__
 !define VER_MAJOR 9
 !define VER_MINOR 1
+!define PATCHLEVEL 0906
```

Comment:
That patchlevel seems a bit random? Should it be zero instead?

Reply:
Is it better to remove this file from the repository?
However, we must update the vim-win32-installer repository.

Reply:
Yes, I was going to delete it, since it is created automatically anyway, but then I remembered that somewhere here it was discussed that template files should also be in the repository.
If Christian doesn't mind, I'll be happy to delete this file.
I have made a pull request for the “vim-win32-installer” repository with the current changes. Please take a look, Takata-san.

Reply:
I thought that not everyone and not always can use the make-file to create the installer, so it is better to keep the template file "gvim_version.nsh".